securitycenter/noxfile.py

```diff
@@ -138,26 +138,3 @@ def cover(session):
     session.run("coverage", "report", "--show-missing", "--fail-under=100")
 
     session.run("coverage", "erase")
-
-
-@nox.session(python=["2.7", "3.5", "3.6", "3.7"])
-def snippets(session):
```

Comment:
@crwilcox Is this something we should generalize and add to the `noxfile.py` template in synthtool?

Reply:
@busunkim96 We don't actually have snippets in most of the APIs (only `bigquery`, `bigtable`, `logging`, and `storage` have them).